src/components/slider/slider.tsx

```diff
@@ -647,7 +662,7 @@ export class Slider implements LabelableComponent, FormComponent, InteractiveCom
         class="graph"
         colorStops={this.histogramStops}
         data={this.histogram}
-        highlightMax={this.isRange ? this.maxValue : this.value}
+        highlightMax={Array.isArray(this.value) ? this.maxValue : this.value}
```

Comment:
I used the array check so that it wouldn't think that highlightMax is being set as an array. Type safety. I can see if I can remove it but there might be some typing issue

Reply:
This won't work right...

<img width="895" alt="image" src="https://user-images.githubusercontent.com/1231455/158712573-8db5b6d4-fff7-4849-891a-5a9f176ea765.png">

Reply:
Problem with `isRange` is that you lose type safety. :\